src/test/java/com/github/dockerjava/cmd/CommitCmdIT.java

```diff
@@ -66,6 +66,11 @@ public void commitWithLabels() throws DockerException {
         assertThat(container.getId(), not(isEmptyString()));
         dockerRule.getClient().startContainerCmd(container.getId()).exec();
 
+        Integer status = dockerRule.getClient().waitContainerCmd(container.getId())
+                .exec(new WaitContainerResultCallback()).awaitStatusCode();
```

Comment:
on one line    client.command()
on other lines fluent calls     .exec()
                                             .await()
That's for eyes redability